src/eva/tests/config/testIodaObsSpaceIASI_Metop-A.yaml

```diff
@@ -0,0 +1,90 @@
+# This YAML file automatically generated by gen_eva_obs_yaml.py
+# from template YAML file: /work2/noaa/da/cmartin/GDASApp/dev/GDASApp/ush/eva/jedi_gsi_compare_rad.yaml
+# on Orion-login-4.HPC.MsState.Edu at 2022-08-31T10:58:39Z
+# template YAML file to create EVA YAML files
+# based on obs spaces listed in JEDI YAML files
+diagnostics:
+- data:
+    type: IodaObsSpace
+    datasets:
+      - name: experiment
+        filenames:
+          - ${data_input_path}/iasi_metop-a.hofx.2021-08-01T00:00:00Z.nc4
```

Comment:
This file is 1.6MB, how many locations is it? Can we subset it some for the test?

Reply:
@CoryMartin-NOAA To be honest, even 1 or 2 obs would be fine for this purpose.  How about I subset to 10 obs so that the file size is similar to AMSU-A?